quickwit-index/src/actors/source/file_source.rs

```diff
@@ -92,10 +105,10 @@ impl AsyncActor for FileSource {
         _message: Self::Message,
         ctx: &ActorContext<Self>,
     ) -> Result<(), ActorExitStatus> {
-        let limit_num_bytes = self.file_position.num_bytes + BATCH_NUM_BYTES_THRESHOLD;
+        let limit_num_bytes = self.counters.previous_offset + BATCH_NUM_BYTES_THRESHOLD;
```

Comment:
I would add short comment on what the method is doing
// collecting batches of documents and sending the batch to the indexer